languages/bash.scm

```diff
@@ -1,2 +1,114 @@
 ; Configuration
 (#language! bash)
+
+; Don't modify string literals
+(string) @leaf
+
+; Allow blank line before
+; TODO Add directives here...
+[
+  (comment)
+  (if_statement)
+] @allow_blank_line_before
+
+; Prepend spaces
+; (Not "surround", to avoid extraneous spaces between delimiters)
```

Comment:
You should not worry about that here. That is the reponsibility of the post-processing.

Reply:
There's/there'll be post-processing? To what degree is it envisioned to operate (e.g., simple whitespace trimming would deal with the extra `\n`s, but targetting delimiters would need to be grammar-aware, etc.)?

Reply:
This?

https://github.com/tweag/topiary/blob/c36d4a2253f337e1a28d497826a84754b8d833f6/README.md?plain=1#L116-L123

What I'm trying to achieve by just prepending spaces is `if foo; then`, rather than `if foo ; then`, for example.

Reply:
Understood. But `if`, `then`, and many more keywords can still be surrounded by spaces.